rakelib/lint.rake

```diff
@@ -0,0 +1,22 @@
+namespace :lint do
+  desc 'Install swiftlint'
+  task :install do |task|
+    swiftlint = `which swiftlint`
```

Comment:
Maybe you forgot the question mark at the end of the method name? (Yes in ruby the question mark at the end of a method name is part of the method name and making methods returning a boolean with a question mark at the end is a convention in ruby)